src/lib/select/select.spec.ts

```diff
@@ -597,6 +596,59 @@ describe('MdSelect', () => {
 
     });
 
+    describe('aria-owns', () => {
+      let fixture: ComponentFixture<ManySelects>;
+
+      beforeEach(() => {
+        fixture = TestBed.createComponent(ManySelects);
+        fixture.detectChanges();
+      });
+
+      it('should set aria-owns properly', () => {
+        const selects = fixture.debugElement.queryAll(By.css('md-select'));
+
+        expect(selects[0].nativeElement.getAttribute('aria-owns'))
+            .toContain('md-select-option', `Expected option IDs to be set in aria-owns attr.`);
+
+        expect(selects[0].nativeElement.getAttribute('aria-owns'))
+            .not.toEqual(selects[1].nativeElement.getAttribute('aria-owns'),
+                `Expected select components to have unique option IDs in aria-owns attr.`);
+      });
```

Comment:
Need a test where you assert that the `aria-owns` contains each of the options's IDs. Something like
```
expect(ariaOwns).toContain(options[0].id);
expect(ariaOwns).toContain(options[1].id);
...
```